src/lib/statsig/statsig.tsx

```diff
@@ -14,6 +14,8 @@ import {useNonReactiveCallback} from '../hooks/useNonReactiveCallback'
 import {LogEvents} from './events'
 import {Gate} from './gates'
 
+const SDK_KEY = 'client-SXJakO39w9vIhl3D44u8UupyzFl4oZ2qPIkjwcvuPsV'
```

Comment:
mind using it below in the Provider? it's hardcoded there too

Reply:
Ah yes, good call